py-polars/polars/dataframe/frame.py

```diff
@@ -1235,6 +1235,26 @@ def __dataframe__(
             )
         return self.to_arrow().__dataframe__(nan_as_null, allow_copy)
 
+    def __dataframe_consortium_standard__(
+        self, /, *, api_version: str | None = None
```

Comment:
I don't think we need the `/` indicator here?

```suggestion
        self, *, api_version: str | None = None
```